numpyro/handlers.py

```diff
@@ -333,27 +334,30 @@ class seed(Messenger):
     .. doctest::
 
        >>> # as context manager
-       >>> with handlers.seed(rng=1):
+       >>> with handlers.seed(rng_key=1):
```

Comment:
This will probably make users confused. Probably we want to keep `rng` as int and `rng_key` as PRNGKey?

Reply:
My plan was removing `rng` argument entirely, and not keep two versions. Maybe we should actually keep this `rng_seed` in that case, and the remaining arguments where we use a `PRNGKey` as `rng_key`?

Reply:
Changed this; `rng` argument is only there for backwards compatibility reasons but is deprecated.

Reply:
That's fine with me.